publish/publish.proj

```diff
@@ -59,6 +59,8 @@
                    SemaphoreBlob="$(Channel)/Binaries/sharedFxPublishSemaphore"
                    Channel="$(Channel)"
                    Version="$(SharedFrameworkNugetVersion)"
+                   SharedFrameworkNugetVersion="$(SharedFrameworkNugetVersion)"
```

Comment:
In this case, they're the same. I didn't see any insurance where they were different, but it brings parity with the previous implementation, so I carried it over